README.md

```diff
@@ -22,6 +22,7 @@ github_ssh_keys:
     - domenukk
     - ldruschk
     - MMunier
+elk_address: ip or url
```

Comment:
I don't think this is used anymore

Reply:
`rg elk_address` yields no results except this one, I think the address is hardcoded everywhere now

Reply:
The question is should it be that way? iirc that was only done to avoid having to debug some issues where the variable wasn't working without spending too much time, in the long run defining it once should be the way to go

Reply:
This setting would have to be synchronized with the terraform file, correct? Is there an easy way to do that?

Reply:
Not the terraform file, currently this is only relevant for the generation of the internal wireguard configuration files.

Reply:
But to answer your question, I am currently not aware of a simple way to do that, but that is primarily caused by me not looking for one yet.

Reply:
Ah right, we don't use the internal networks no more.